tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -423,7 +424,10 @@ private void SetTraceSamplingPriority(Span span)
         private void AddResponseHeaderTags(ITransport transport, Span span)
         {
             TryAddEndPoint(span);
-            var headers = transport.GetResponseHeaders();
+            var headers =
```

Comment:
The one difference I saw between all the Classic/enableSecurity=True and Integrated/enableSecurity=True snapshots is that the blocked spans in the Integrated test have an additional `http.response.headers.content-type: text/html; charset=utf-8` tag. Is it okay if that is missing in classic mode? Or since we are writing the response and we know the content type, perhaps we can include that one tag in the Classic fallback case?

Reply:
It would be better to be able to write the headers in both modes, but we don't rely on it in the backend, it's just something we consider might help clients understand the request we've tagged.

While we know the content-type when we block a request, we also use this method in monitor mode, and I think it's the case where we don't control the content being sent that's more interesting to clients.